test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -3063,9 +3063,9 @@ public string F2(int a)
             }
         }
 
-        // Tests that pack works with -MSBuildVersion set to 12
-        [Fact]
-        public void PackCommand_WithMsBuild12()
+        // Tests that pack works with -MSBuildVersion set to 15.1
+        [Fact(Skip = "Re-enable this when MSBuild 15.1 is installed on CI machines")]
+        public void PackCommand_WithMsBuild151()
```

Comment:
We really need a Trait that defines tests that **must** run when on the CI vs tests that can run for users if they have that version of MSbuild installed.